src/modules/todos.js

```diff
@@ -58,11 +105,35 @@ export const markToDo = (e, todos) => {
   renderToDoList(todos);
 };
 
+/* 
+const markToDo = (e, todos) => {
+  const clickedCheckbox = e.target.closest('.todo-checkbox');
+  const clickedToDo = clickedCheckbox. nextElementSibling;
+  const toDoIndex = todos.findIndex(
+    (toDoItem) => toDoItem.toDoItem === clickedToDo.value,
+  );
+  todos = getTodo();
+  todos[toDoIndex].completed = !todos[toDoIndex].completed;
+  setToDo(todos);
+  renderToDoList(todos);
+}
+*/
+
```

Comment:
Please delete commented code inside your code because it makes your code unclean. 😉